src/main/java/org/ambraproject/rhino/service/taxonomy/impl/TaxonomyClassificationServiceImpl.java

```diff
@@ -122,12 +120,13 @@
   @Autowired
   protected HibernateTemplate hibernateTemplate;
 
+  private static final String[] CATEGORY_BLACKLIST =
+      new String[] {"/Earth sciences/Geography/Locations/"};
```

Comment:
See https://jira.plos.org/jira/browse/AMEC-100. You can also look in plos/graveyard under ambra to see the ancient history of this feature. Basically it was a one-time hack which may or may not still be needed. But at some point the hack of blacklisting a single category turned into a configurable feature. It seemed to me that this was absolutely not something that needed to be configurable at runtime, so I just moved it here. It's not clear if this is still a feature we need, but after talking with Jack he felt it was best to keep the feature in for now.

Reply:
Interesting, thanks!  Would it be worth noting the ticket in a comment?

Reply:
Good point, will do.

Reply:
4f6e354c